### PR TITLE
Complete section download and section testimonials-mobile

### PR DESCRIPTION
**Pull request by ronlechiht**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -66,6 +66,89 @@ <h4 class="sub-heading article-title">organized layers</h4>
...
         </div>
       </section>
+
+      <!--================================= Section Download  =================================-->
+      <section class="section-download">
+        <h2 class="main-heading">get your free version</h2>
+        <p class="download-description">Having used discount toner cartridges for twenty years, there 
+          have been a lot of changes in the toner cartridge market. The market today is approximately 
+          a twenty billion dollar business.</p>
+        <button class="btn btn-download"><a href="#">download</a></button>
```
Comment (bantla) [inline src/index.html:76]: Just to share information. The download button here is called CTA button in landing pages. You can use btn-cta to name buttons like that. You can see more [here](https://unbounce.com/landing-page-articles/what-is-a-cta/)

Comment (bannguyenasnet) [inline src/index.html:76]: Just to share information. The download button here is called CTA button in landing pages. You can use btn-cta to name buttons like that. You can see more [here](https://unbounce.com/landing-page-articles/what-is-a-cta/)

File: src/index.html [hunk 2]
```diff
@@ -66,6 +66,89 @@ <h4 class="sub-heading article-title">organized layers</h4>
...
+      </section>
+
+      <!--================================= Section Testimonials =================================-->
+      <section class="section-testimonials">
+        <h2 class="main-heading">testimonials</h2>
+        <div class="customers-table">
+          <div class="customer">
+            <img src="/src/assets/images/section-testimonials/person1.jpg" alt="lulu hoffman" class="customer-img">
```
Comment (bannguyenasnet) [inline src/index.html:84]: You should add circle css styles for avatars

File: src/styles/sections/section-testimonials.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  margin-top: 3px;
+}
+
+.customer-job {
+  font-size: 12px;
+  color: #7d7d7d;
+  margin-top: 9px;
+}
+
+.customer-rectangle {
```
Comment (bannguyenasnet) [inline src/styles/sections/section-testimonials.css:29]: customer-rectangle can be named as divider. You can see more [here](https://www.w3schools.com/howto/howto_css_dividers.asp)

File: src/styles/sections/section-testimonials.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.customer-facebook {
+  width: 18px;
+  height: 18px;
+  background: url("../../assets/images/section-testimonials/testimonials-social.png") 0 0;
+}
+
+.customer-linkedin {
+  width: 18.45px;
```
Comment (bannguyenasnet) [inline src/styles/sections/section-testimonials.css:55]: Can you round the width? Should round values likes that if possible

File: src/index.html [hunk 5]
```diff
@@ -66,6 +66,89 @@ <h4 class="sub-heading article-title">organized layers</h4>
...
+            <img src="/src/assets/images/section-testimonials/person3.jpg" alt="person3" class="customer-img">
+            <div class="customer-info">
+              <h4 class="sub-heading customer-name">dustin gonzales</h4>
+              <h4 class="customer-job">CEO at hofmannui.net</h4>
+              <div class="customer-rectangle"></div>
+              <p class="customer-comment">Do you remember the story of "Thw Wizard of Oz"? In this story we saw the 
+                characters of Dorothy, who had wanted to run away from home because she 
+                thought there was a better place somewhere.</p>
+              <ul class="customer-social">
+                <li class="social-item"><a href="#"><div class="customer-facebook"></div></a></li>
```
Comment (bannguyenasnet) [inline src/index.html:125]: You should use <i> tag for icons. See more [here](https://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span)

Comment (bannguyenasnet) [inline src/index.html:125]: You should use `i` tag for icons. See more [here](https://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span)

**Pull request merged**